public/javascripts/common/wrapperFrame.js

```diff
@@ -0,0 +1,53 @@
+const iframe = document.getElementById('wrapperFrame');
```

Comment:
There are just a few places throughout the file that could use some helpful comments since the bulk of the work is happening here! Makes sense while you're quickly iterating, but it's probably about time to comment up!

Reply:
Definitely agreed! It needs much more comments.